doc/manual/nuc_handbook.rst

```diff
@@ -0,0 +1,257 @@
+NixOS for ARTIQ (NUC handbook)
+==============================
+
+ARTIQ is a fully open-source project, and can be installed and used on any operating system. All else being equal, however, M-Labs suggests using the Linux distribution `NixOS <https://nixos.org/>`_, based around the Nix package manager, which ARTIQ is also distributed through. NixOS is used in-house by M-Labs to build, test and develop ARTIQ; if you have total freedom of OS and you want to quickly set up an ARTIQ environment that will *definitely work*, NixOS should be your first choice. Even if you are familiar with Linux, it has some peculiarities, but the advantages it offers (reproduceability, reliability, stability) are worth the learning curve.
```

Comment:
> ARTIQ [...] can be installed and used on any operating system.

As much as everyone appreciates good documentation, that's obscenely wrong.
The entire line of argument appears pointless and presumptuous for a "handbook". After all, the explicit assumption is that the intended audience already made the choice.
And what does it mean that most of this handbook doesn't address ARTIQ specifics at all?

Reply:
> The explicit assumption is that the intended audience already made the choice.

There's recently been discussion that some people would appreciate instructions on how to get ARTIQ working from a completely fresh operating system up. The idea was to cover that use case as well, as opposed to only people purchasing NUCs. It should be relatively trivial to strip that implication out if it's incorrect.

Reply:
> any operating system.

This is effectively a typo, there were meant to be more qualifiers. Embarrassing for me. Will fix it together with other suggestions (not on the correct machine at this time.)